rfc8446.xml

```diff
@@ -6412,7 +6423,9 @@ from older TLS implementations.</t>
 ]]></artwork></figure>
 
 <t>Values within "obsolete_RESERVED" ranges are used in previous versions
-of TLS and MUST NOT be offered or negotiated by TLS 1.3 implementations.
+of TLS and MUST NOT be offered by TLS 1.3 implementations.
+Values marked "_RESERVED" are not defined for use in TLS 1.3 and MUST NOT
+be negotiated by TLS 1.3 implementations.
```

Comment:
These two chunks I'm not sure we need to take, but potentially resolve the brainpool peoples' sadness (by allowing you to send them in a 1.3 handshake for use if 1.2 is negotiated).